bindinfo/session_handle.go

```diff
@@ -70,6 +70,12 @@ func (h *SessionHandle) AddBindRecord(record *BindRecord) error {
 	return err
 }
 
+// DropBindRecord drops a BindRecord in the cache.
+func (h *SessionHandle) DropBindRecord(record *BindRecord) {
+	hash, meta := newBindMetaWithoutAst(record)
+	h.ch.removeDeletedBindMeta(hash, meta)
```

Comment:
According to the discussion, if the bind exists, we should change it's status to deleted, and if it does not exist, we should insert one deleted bind.

Reply:
But this is session binding, just delete it will be ok.

Reply:
We need to make sure it will not be affected by global bindings.

Reply:
Let me fix.